.gitignore.dist

```diff
@@ -0,0 +1,13 @@
+.DS_Store
+.vscode
+Thumbs.db
+phpcs.xml
+phpunit.xml
+wp-cli.local.yml
+node_modules/
+*.sql
+*.tar.gz
+*.zip
+
+# Client build files
```

Comment:
Only sanity-test item difference between this and the `built` branch .gitignore is that the build branch also excludes `.npm`, which seems like a fine thing to ignore in both the .gitignore and .gitignore.dist files... might be worth adding.